gatling-recorder/src/main/scala/io/gatling/recorder/scenario/template/RequestTemplate.scala

```diff
@@ -51,10 +51,12 @@ private[scenario] object RequestTemplate {
         }.getOrElse("")
 
       def renderLongString(value: String) =
-        if (value.nonEmpty)
-          value.grouped(65535).map(protectWithTripleQuotes).mkFastring(" + ")
-        else
-          protectWithTripleQuotes("")
+        if (value.size > MaxLiteralSize) {
+          wrapLongString(value.grouped(MaxLiteralSize).map(protectWithTripleQuotes).mkFastring(", "))
+        } else
+          protectWithTripleQuotes(value)
+
+      def wrapLongString(value: Fastring) = "joinStrings(" + value + ")"
```

Comment:
I've checked, even if I split large String as ("part1" + "part2" + "part3")  onto small fragments, I still have compilation error below if total fragments length exceeds 65534: 
_Error:scalac: Error while emitting TestLargeStrings$
Maximum String literal length exceeded_

To use Seq.mkString is good idea, will update the request